kustomize/kustomization.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- ./binding-apiserver
+- ./apiserver
+- ./clustersynchro-manager
+- ./controller-manager
+- ./crds
+- ./internalstorage
```

Comment:
We can only choose one storage component when deploying, so we can't specify internalstorage here, but rather internalstorage/mysql or internalstorage/postgres